vuepress/docs/next/docs/reference/backup-restore.md

```diff
@@ -0,0 +1,87 @@
+---
+sidebarDepth: 2
+---
+target=restored
+original=source
+# Backup and Restore an Entando Application
+
+This guide provides general instructions on how to backup and restore an existing Entando Application. The process entails the migration of databases, filesystem data, Keycloak instance, and bundles, plus the reconfiguration of the service address. 
+
+### Initial Considerations to Note
+* The restored instance can be on another cluster or a different namespace of the same cluster hosting the original Entando instance. 
+* The source and restored instances cannot be executed simultaneously, and data cannot be shared between them.
+* Bundles are externally deployed to an Entando Application and therefore must be reinstalled after the new instance is built. More details are below.
```

Comment:
"Bundles are externally deployed to an Entando Application " -> "Bundles are deployed via Kubernetes to a specific Entando Application"